### PR TITLE
Updated date and location

### PR DESCRIPTION
**Pull request by jakobreinwald**
Updated date and location

### CODE REVIEW

File: src/components/HomePage/FAQSection.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ AccordionQA.propTypes = {
 function FAQSection() {
 	const hothNum = 'XI';
 	const appDeadline = 'Monday, February 26, 2024';
-	const hothDate = 'Saturday, March 2, 2024';
+	const hothDate = 'Sunday, March 3, 2024';
 	const hothLocation = 'Hill';
```
Comment (maggieelli) [inline src/components/HomePage/FAQSection.js:116]: Should also change hothLocation

File: src/components/HomePage/Banner.js [hunk 2]
```diff
@@ -210,7 +210,7 @@ function Banner() {
 				</NoSsr>
 
 				<Typography variant='h5' className={classes.text} component='h3'
-					style={{ fontWeight: 500, marginTop: 10 }}><b>Location:</b> The Hill, room TBD
+					style={{ fontWeight: 500, marginTop: 10 }}><b>Location:</b> Carnesale Commons, Palisades Room
```
Comment (maggieelli) [inline src/components/HomePage/Banner.js:213]: This is a nit but I think the banner would look a lot better if this text was right-aligned

**Review by maggieelli — COMMENTED** [review]
Small changes requested but otherwise looks good!

File: src/components/HomePage/Banner.js (hunk 2 above)
Comment (jakobreinwald) [reply src/components/HomePage/Banner.js:213]: You're totally right

**Review by maggieelli — CHANGES_REQUESTED** [review]
One more change I missed last time - in src/components/HomePage/HothDescription.js line 116, it also says room TBD still

**Review by maggieelli — APPROVED** [review]
LGTM! :)

**Review by nathanzzhang — APPROVED** [review]
lgtm!!

**Pull request merged**